mrtarget/common/ElasticsearchLoader.py

```diff
@@ -89,17 +89,13 @@ def get_versioned_index(index_name, check_custom_idxs=False):
 
         return idx_name + suffix
 
-    def is_dry_run(self):
-        return self.dry_run
-
     def put(self,
             index_name,
             doc_type,
             ID,
             body,
             create_index = True,
             operation= None,
-            routing = None,
```

Comment:
Removing just one option from the list might cause problems if this is called by argument position anywhere in the code base.

Reply:
Done!